.github/workflows/ci.yaml

```diff
@@ -33,3 +33,10 @@ jobs:
                   (cd /tmp && git clone https://github.com/anuvu/squashfs && cd squashfs && make && sudo cp squashtool/squashtool /usr/bin)
             - run: |
                   make check STORAGE_TYPE=${{ matrix.storage-type }} PRIVILEGE_LEVEL=${{ matrix.privilege-level }}
+            - if: github.event_name == 'release' && github.event.action == 'published'
+              name: Publish artifacts on releases
+              uses: Roang-zero1/github-upload-release-artifacts-action@master
+              env:
+                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+              with:
+                  args: stacker*
```

Comment:
Can you add a description of what this actually does to the commit message? In particular, I don't think we want to publish "stacker-dynamic", just the final "stacker" binary produced by the build system.